tests/keras_tests/feature_networks_tests/feature_networks/bn_attributes_quantization_test.py

```diff
@@ -0,0 +1,170 @@
+        # we add a relu BEFORE the BN layer to prevent it from folding, so we can test its quantization
+        x = layers.ReLU()(x)
+        x = layers.BatchNormalization(beta_initializer='glorot_uniform',
+                                      gamma_initializer=tf.keras.initializers.RandomUniform(minval=0.0001, maxval=1.05),
+                                      moving_mean_initializer='glorot_uniform',
+                                      moving_variance_initializer=tf.keras.initializers.RandomUniform(minval=0.0001,
+                                                                                                      maxval=1.05))(x)
+        return tf.keras.models.Model(inputs=inputs, outputs=x)
+
+    def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
+        float_bn_layer = get_layers_from_model_by_type(float_model, layers.BatchNormalization, include_wrapped_layers=False)
+        self.unit_test.assertTrue(len(float_bn_layer) == 1, "Expecting the float model to have exactly 1 BN layer")
+        float_bn_layer = float_bn_layer[0]
+
+        quant_bn_layer = get_layers_from_model_by_type(quantized_model, layers.BatchNormalization, include_wrapped_layers=True)
+        self.unit_test.assertTrue(len(quant_bn_layer) == 1, "Expecting the quantized model to have exactly 1 BN layer")
+        quant_bn_layer = quant_bn_layer[0]
+
+        # Verify that the BN layer is wrapped with weights quantization wrapper
+        self.unit_test.assertTrue(isinstance(quant_bn_layer, KerasQuantizationWrapper),
+                                  "BN layer is supposed to be wrapped with a weights quantization wrapper")
+
+        # Verify BN attributes quantization
+        f_bn_weights = float_bn_layer.weights
+        q_bn_weights = quant_bn_layer.get_quantized_weights()
+
+        f_beta = [w for w in f_bn_weights if BETA in w.name]
+        self.unit_test.assertTrue(len(f_beta) == 1, "Expecting float model BN layer to have a BETA attribute")
+        f_beta = f_beta[0]
+        q_beta = q_bn_weights.get(BETA)
+        self.unit_test.assertTrue(q_beta is not None, "Expecting quantized model BN layer to have a BETA attribute")
+        self.unit_test.assertTrue(np.any(f_beta != q_beta), "Float and quantized BETA attributes are expected to have different values")
+
+        f_gamma = [w for w in f_bn_weights if GAMMA in w.name]
+        self.unit_test.assertTrue(len(f_gamma) == 1, "Expecting float model BN layer to have a GAMMA attribute")
+        f_gamma = f_gamma[0]
+        q_gamma = q_bn_weights.get(GAMMA)
+        self.unit_test.assertTrue(q_gamma is not None, "Expecting quantized model BN layer to have a GAMMA attribute")
+        self.unit_test.assertTrue(np.any(f_gamma != q_gamma),
+                                  "Float and quantized GAMMA attributes are expected to have different values")
+
+        # Verify output difference (sanity)
+        y = float_model.predict(input_x)
```

Comment:
I think the predictions part is not needed (and opens a door for randomness issues).